### ISSUE TITLE
I18n support

### ISSUE THREAD

**Issue by zedtux** [michealparks/galeri-www#2]
It would be great to be able to have the artwork labels in the translated version when available.

**Comment by zedtux** [comment]
Is this project abandoned?

**Comment by michealparks** [comment]
@zedtux Nope! I regret that I haven't had much time to work on it this year but I expect to deploy a big release in a few weeks. I18n support isn't planned yet since - to my knowledge - no museums that we pull from have data in other languages. By the way, you can track development [here](https://github.com/michealparks/galeri).
